File: server/src/routes/health.js
// server/src/routes/health.js
import express from 'express'
import { cache } from '../services/cache.js'

export const healthRouter = express.Router()

const startTime = Date.now()

healthRouter.get('/live', (req, res) => {
  res.json({ status: 'ok', uptime: Math.floor((Date.now() - startTime) / 1000) })
})

healthRouter.get('/', (req, res) => {
  res.json({
    status: 'healthy',
    uptime: Math.floor((Date.now() - startTime) / 1000),
    cache:  cache.getStats(),
    version: '1.0.0',
  })
})

File: server-py/app/routes/health.py
# server-py/app/routes/health.py
import time

from fastapi import APIRouter

from app.services.cache import cache

router = APIRouter()

_start_time = time.time()


@router.get("/live", summary="服务健康检查",)
async def live():
    return {"status": "ok", "uptime": int(time.time() - _start_time)}


@router.get("/")
async def health():
    return {
        "status": "healthy",
        "uptime": int(time.time() - _start_time),
        "cache": cache.get_stats(),
        "version": "1.0.0",
    }
